ClinGen/clincoded
# Connects to #772. Connects to #796. Adding Bustamante data integration.

This PR consists of changes pertinent to the integration of Bustamante API data in the "ClinGen Predictors" table on the computational tab.

**Known issues:**
1. Not all variants are returning data from Bustamante API.
2. Not all variants are returning "CFTR" score data from Bustamante API.

**Technical details:**
1. This PR also includes #796, for which we are suppressing the display of "CFTR" row in the table if the CFTR object is not returned in the Bustamante API response.

**Steps to test:**
1. Login and enter ClinVar ID 55847 or 36484 at the selection interface.
2. Proceed to the "Predictors" tab and expect to see only the REVEL table row present in the "ClinGen Predictors" table. And the "Score" column has score data.
3. Now try ClinVar ID 139214 and expect the "Score" column has no data but a "No data found" string.
